app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -248,11 +307,86 @@ export default class AMRHeatmapVis extends React.Component {
         onNodeHoverOut: this.onNodeHoverOut,
         onColumnLabelClick: onSampleLabelClick,
         onRowLabelClick: this.onRowLabelClick,
+        columnMetadata: this.getSelectedMetadataFields(), // gets the selected metadata *fields*
+        onColumnMetadataLabelMove: this.onNodeHoverMove,
+        onColumnMetadataLabelHover: this.onMetadataLabelHover,
+        onColumnMetadataLabelOut: this.onMetadataLabelOut,
+        onAddColumnMetadataClick: this.onMetadataAddButtonClick,
+        marginLeft: 75, // our gene names are very short, so this is to prevent metadata names from disappearing
```

Comment:
Not sure you need this. I think the `rowLabelsWidth` in `Heatmap` already takes the metadata name lengths into account.

Reply:
It does...until you click on them to sort the samples and the metadata names get cut off.

Reply:
Ahh I see, good catch. The sort icon adds some additional width. Will "50" work? (it might based on the width of the sort icon plus the sort icon's left margin). Don't want to add extra unnecessary margin.

Reply:
Changed it to 50